foundations/introduction/asking_for_help.md

```diff
@@ -1,58 +1,58 @@
 
-#### Ask About the Problem at Hand, Not the Solution Itself
+If your question does not provide any code or other relevant information, it will be difficult for others to help you. In order to fully understand your question, there will be a lot of unnecessary back and forth conversation. This can be discouraging for you because any answers with incomplete information will not solve your problem. This process can be frustrating for those helping because the meaning of your original question changes as you add more context to it. If you truly want to ask a conceptual question, you should indicate this as part of your question.
+
+#### Ask about the problem at hand, not the solution itself
+
+A lot of learners ask exactly how to approach a given task or assignment in this curriculum, such as the below:
 
-A lot of learners ask exactly how to approach a given task or assignment in this curriculum, such as the below: 
- 
 -   “How do I complete Step 5 of the Rock Paper Scissors Assignment”?
 
-Keep in mind that you’re actually supposed to figure out how to solve the assignments yourself, and figuring out an approach is essential to your learning journey. A much better question might look like this: 
- 
--   “Hey, I’m trying to return a string that displays the winner in Rock Paper Scissors, but I’m getting a syntax error on line 12. How can I fix this? Here’s my code.” 
+Keep in mind that you’re actually supposed to figure out how to solve the assignments yourself, and figuring out an approach is essential to your learning journey. A much better question might look like this:
+
+-   “Hey, I’m trying to return a string that displays the winner in Rock Paper Scissors, but I’m getting a syntax error on line 12. How can I fix this? Here’s my code.”
 
 In sharing your attempt, people know what you’ve tried and won’t suggest things that might not have worked out for you. It also allows them to debug your current iteration of code, rather than sending you down a path that has you start all over again when you might be very close to a solution.
 
 Now, if you’re completely stuck with where to start, it’s completely fine to let people know that you’re stuck. Asking where you can start and what you can research to get on the right track empowers you to be able to resolve issues largely on your own in the future, and might even empower you to help others with the same issue later on. It is also recommended that you share your pseudo code so people can nudge you in the right direction or correct any misunderstandings you may have.
 
-#### Don’t Take Asking for More Context to Heart
+#### Don’t take asking for nore context to heart
```

Comment:
```suggestion
#### Don’t take asking for more context to heart
```

Reply:
Great catch! I must have done that it too much of a hurry!